<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptEditorClass</class>
 <widget class="QMainWindow" name="ScriptEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScriptEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>脚本编辑</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>脚本窗口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>LUA代码窗口</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTextEdit" name="scrEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="ParseButton">
            <property name="text">
             <string>&gt;&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="10,0,4">
            <item>
             <widget class="QTextEdit" name="luaEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>输出</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="retEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="RunButton">
          <property name="text">
           <string>运行</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>UI测试</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>图片</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="4" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>文件名:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="ImageFileNameEdit"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>渐变时间:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="universalTimeEdit"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>X位置:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="XEdit"/>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Y位置:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLineEdit" name="YEdit"/>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>显示图片</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="EffectsLayerButton">
             <property name="text">
              <string>效果层</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="MidLayerButton">
             <property name="text">
              <string>中图层</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="LeftLayerButton">
             <property name="text">
              <string>左图层</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QRadioButton" name="RightLayerButton">
             <property name="text">
              <string>右图层</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="BGLayerButton">
             <property name="text">
              <string>背景层</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>对话框</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>对话框属性</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>X位置:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="DialogXEdit"/>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>文本左边距:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLineEdit" name="MarginLeftEdit"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Y位置:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="DialogYEdit"/>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>文本顶边距:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLineEdit" name="MarginTopEdit"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>宽度:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="DialogWidthEdit"/>
              </item>
              <item row="2" column="2">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>文本宽度:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QLineEdit" name="TextWidthEdit"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>高度:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="DialogHeightEdit"/>
              </item>
              <item row="3" column="2">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>文本高度:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="3">
               <widget class="QLineEdit" name="TextHeightEdit"/>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>字体:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="FontEdit"/>
              </item>
              <item row="6" column="0" colspan="4">
               <widget class="QPushButton" name="SetTextColorButton">
                <property name="text">
                 <string>设置文字颜色</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1" rowspan="3">
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>消息测试</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLineEdit" name="RoleNameEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="ShowRoleNameButton">
                <property name="text">
                 <string>显示角色名</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>延时(秒):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="DelayEdit">
                <property name="text">
                 <string>0.0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextEdit" name="TextEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="ShowTextButton">
                <property name="text">
                 <string>显示文字</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="ClearTextButton">
                <property name="text">
                 <string>清空</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>角色名称框属性</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>X位置:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="RoleNameXEdit"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Y位置:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="RoleNameYEdit"/>
              </item>
              <item row="3" column="0" colspan="5">
               <widget class="QPushButton" name="SetRoleNameColorButton">
                <property name="text">
                 <string>设置角色名颜色</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>宽度:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="RoleNameFontEdit"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>字体:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLineEdit" name="RoleNameWidthEdit"/>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>高度:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLineEdit" name="RoleNameHeightEdit"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="SetDialogButton">
             <property name="text">
              <string>设置对话框</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="scripteditor.qrc"/>
 </resources>
 <connections/>
</ui>
